content/en/docs/setup/install/multicluster/primary-remote/index.md

```diff
@@ -128,6 +128,10 @@ $ istioctl x create-remote-secret \
 
 Save the address of `cluster1`’s east-west gateway.
 
+{{< tip >}}
+If you are testing primary-remote multicluster setup on `Kind` you can use [MetalLB](https://metallb.universe.tf/installation/) to make use of `EXTERNAL-IP` for `LoadBalancer` services.
```

Comment:
I believe they do not uppercase the `k` in kind.

Reply:
Done.